editor/plugins/script_editor_plugin.cpp

```diff
@@ -3226,6 +3226,17 @@ void ScriptEditor::input(const Ref<InputEvent> &p_event) {
 			}
 		}
 	}
+
+	const Ref<InputEventKey> key = p_event;
+	if (key.is_valid() && key->is_pressed() && is_visible_in_tree()) {
+		if (key->get_keycode() == Key::BACK) {
+			_history_back();
+		}
+
+		if (key->get_keycode() == Key::FORWARD) {
+			_history_forward();
+		}
+	}
```

Comment:
I wonder if this can be enabled via (configurable) shortcuts instead of hardcoded. I see that for the extra mouse buttons this is hardcoded here with some justifications, but I don't know if the same applies for those key events.

CC @KoBeWi

Reply:
I looked into a similar approach at first but it looks like we already have defined shortcuts, and we don't have the ability to record several keys combinations for the same shortcut.

Reply:
Shortcuts do support multiple events, e.g. we have Ctrl+Shift+Z and Ctrl+Y for redo.

Reply:
@KoBeWi Can you point to an example code? I can update the PR to use the shortcut approach instead.

Reply:
ED_SHORTCUT_ARRAY
https://github.com/godotengine/godot/blob/f648de1a83cf006dbfdaa075219ad4348628e58f/editor/plugins/canvas_item_editor_plugin.cpp#L5263-L5264

Reply:
Updated to use `ED_SHORTCUT_ARRAY`!